developer-docs-site/docs/issues-and-workarounds.md

```diff
@@ -5,14 +5,107 @@ slug: "issues-and-workarounds"
 
 # Issues and Workarounds
 
-This page documents issues that frequently come up and the suggested workarounds. 
+This page provides workarounds and answers for issues and questions that frequently come up. 
 
 :::tip Help keep this page up-to-date
 If you found an issue that is not on this page, submit a [GitHub Issue](https://github.com/aptos-labs/aptos-core/issues). Make sure to follow the issue format used in this document. 
 :::
 
 ## Nodes
 
+### Invalid EpochChangeProof: Waypoint value mismatch
+
+#### Description
+
+Run a validator node and receive this error:
+
+```json
+{"error":"Invalid EpochChangeProof: Waypoint value mismatch: waypoint value = 3384a932349524093cda8cea714691e668d668fb34260d8a5f77c667d7724372, given value = 81ee9bd880acd25ad617e55913b7345dc01b861adf43971259a22e9a5c82315c","event":"error","name":"initialize"}
+```
+
+#### Workaround
+
+Delete the `secure-data.json` file because very likely you are using an older version of it. See [Bootstrapping validator node](nodes/validator-node/operator/connect-to-aptos-network.md#bootstrapping-validator-node) for the location of this file. 
+
+### How to find out when the next epoch starts
+
+  :::tip Current epoch duration
+  The Aptos current epoch duration is 1 hour.
+  :::
```

Comment:
This is not rendering properly. Maybe remove the indentation?

Reply:
Done.